lib/vanagon/engine/pooler.rb

```diff
@@ -7,7 +7,7 @@ class Pooler < Base
       attr_reader :token
 
       # The vmpooler_template is required to use the pooler engine
-      def initialize(platform, target = nil)
+      def initialize(platform, target = nil, opts = {})
```

Comment:
@mckern since none of these actually use specific kwargs this would be a case where I would just `**kwargs` in python to avoid having to update all these signatures when we add new options they don't care about. What should I do in ruby?